velox/dwio/common/FileSink.h

```diff
@@ -36,13 +40,15 @@ class FileSink : public Closeable {
     memory::MemoryPool* pool{nullptr};
     MetricsLogPtr metricLogger{MetricsLog::voidLog()};
     IoStatistics* stats{nullptr};
+    std::shared_ptr<const Config> properties{nullptr};
```

Comment:
Shall we explicitly call this connectorProperties and comment? Otherwise, can we move bufferWrite into properties? thanks!

Reply:
I went with connectorProperties and comment. Since properties are immutable, we cannot move bufferWrite.